Formula/aws-sessionmanager.rb

```diff
@@ -0,0 +1,19 @@
+class AwsSessionmanager < Formula
+  desc "Session Manager Plugin for the AWS CLI"
+  homepage "https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with.html"
+  url "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/mac/sessionmanager-bundle.zip"
+  version "1.0.0.0"
```

Comment:
Upstream says version is `1.0.0`

Reply:
The version string is actually stored in a `VERSION` file that states that version